lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -252,6 +228,19 @@ class UserProfileBloc {
         localizedReason: action.localizedReason));
   }
 
+  Future _getEnrolledBiometrics(GetEnrolledBiometrics action) async {
```

Comment:
It is a bit strange that this action is called _getEnrolledBiometrics and updates the model.
I think that we should also separate that settings in the model from what we actually show to the user.
For example the isFingerprintEnabled can stay true if the user has set it like this even when we detect that the biometrics is not enabled. In the settings UI we can take that into consideration and show it as disabled.